### PR TITLE
Feature/entity level filtering on display

### LINKED ISSUE #82: Add criteria for filtering the level of entities to be displayed

### PR DESCRIPTION
**Pull request by flegastelois**
Close #82

### CODE REVIEW

File: inc/baseclass.class.php [hunk 1]
```diff
@@ -151,6 +152,6 @@ function __construct($config = array()) {
       $this->sql_date_closed = PluginMreportingCommon::getSQLDate("glpi_tickets.closedate",
                                                                   $config['delay'],
                                                                   $config['randname']);
-
+
```
Comment (trasher) [inline inc/baseclass.class.php]: No really relevant :D

File: inc/common.class.php [hunk 2]
```diff
@@ -1467,6 +1467,98 @@ static function selectorDateinterval() {
       Html::showDateFormItem("date2".$randname, $date2, false);
    }
 
+   /**
+    * Show entity level selector.
+    * @param  none
```
Comment (trasher) [inline inc/common.class.php]: Not sure this is a standard way, if there is no param, just remove the line.

Comment (flegastelois) [inline inc/common.class.php]: Ok, done.

**Comment by flegastelois** [comment]
> Could you also please squas all commits into one?

If you provide me the best way to do that, yes I can.

**Comment by trasher** [comment]
@flegastelois just `git rebase -i develop` and set all commits but the first to "squash" :)

**Pull request closed without merging**